apps/omg_eth/lib/eth.ex

```diff
@@ -35,9 +35,8 @@ defmodule OMG.Eth do
 
   def get_ethereum_height do
     case Ethereumex.HttpClient.eth_block_number() do
-      {:ok, "0x" <> height_hex} ->
-        {height, ""} = Integer.parse(height_hex, 16)
-        {:ok, height}
+      {:ok, height_hex} ->
+        {:ok, int_from_hex(height_hex)}
```

Comment:
a bit confusing, using import, ```Encoding.int_from_hex``` it is more direct

Reply:
there's a lot of occurrences in this file and `Encoding` is an `Eth` internal tool. There would be a lot of `Encoding...`. Then, by consistency, we'd flood `Eth.RootChain` which these `Encoding` too.

In general I prefer explicit, but in this situation I think no prefix is more readable